guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Renewing_a_Custom_SSL_Certificate_{smart-proxy-context}_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+Do not use the same command on more than one {SmartProxyServer}.
+
+.Prerequisites
+* You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+.Procedure
+. On {ProjectServer} validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c _/root/{smart-proxy-context}_cert/New_{smart-proxy-context}_cert.pem_ \
+-k _/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_ \
+-b _/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_
+----
++
+If you set a wildcard value `*` for the certificate’s Common Name `CN =` in the `/root/{smart-proxy-context}_cert/openssl.cnf` configuration file, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
```

Comment:
Where does this come from? Do we document setting this wildcard somewhere?

Reply:
```suggestion
If the CN has a wildcard value, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
```

Otherwise I think this will confuse users. The use of `/root/capsule_cert/openssl.cnf` is *only* if a user follows the procedure for creating their own custom certificates locally (which is likely a very rare scenario). But users could create wildcard certificates through their provider as well and this option is needed in any of those cases.

Reply:
Applied your suggested change :+1: